### ISSUE TITLE
WCAG2ICT Project: Update to Guidelines

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#23]
- [x] Sanity check that none of the language in the WCAG 2.2 Guidelines has changed
- [x] Propose content for the new guideline 2.5 Input Modalities
- [x] Add the new guideline section 2.5 Input Modalities (originating from WCAG 2.1) and changes to other guidelines into the Editor's draft

**Comment by maryjom** [comment]
### Guideline 1.3 Adaptable
From [Guideline 1.3](https://www.w3.org/TR/WCAG22/#adaptable):
> Create content that can be presented in different ways (for example simpler layout) without losing information or structure.

#### Additional Guidance When Applying Guideline 1.3 to Non-Web Documents and Software:
In WCAG 2.2, the Guidelines are provided for framing and understanding the success criteria under them but are not required for conformance to WCAG. Guideline 1.3 applies directly as written.

Intent from Understanding Guideline 1.3 in Understanding WCAG 2.2 ([View collapsible version of guidance for Guideline 1.3](https://www.w3.org/WAI/WCAG22/Understanding/adaptable#intent)):

>The purpose of this guideline is to ensure that all information is available in a form that can be perceived by all users, for example, spoken aloud, or presented in a simpler visual layout. If all of the information is available in a form that can be determined by software, then it can be presented to users in different ways (visually, audibly, tactilely etc.). If information is embedded in a particular presentation in such a way that the structure and information cannot be programmatically determined by the assistive technology, then it cannot be rendered in other formats as needed by the user.

**Comment by maryjom** [comment]
## Updated in WCAG 2.1 was Guideline 2.3

### Guideline 2.3 Seizures and Physical Reactions
> Do not design content in a way that is known to cause seizures or physical reactions.

#### Additional Guidance When Applying Guideline 2.3 to Non-Web Documents and Software:
In WCAG 2.0, the Guidelines are provided for framing and understanding the success criteria under them but are not required for conformance to WCAG. Guideline 2.3 applies directly as written.

Intent from [Understanding Guideline 2.3 in Understanding WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/navigable) ([View collapsible version of guidance for Guideline 2.3](http://www.w3.org/WAI/WCAG22/Understanding/navigable#intent)):
**NOTE: The understanding's Intent for Guideline 2.3 is unchanged from previous versions and a WCAG issue was opened to address that.**


## Updated in WCAG 2.1 was the Intent section of Understanding Guideline 2.4 Navigable

The intent of this guideline is to help users find the content they need and allow them to keep track of their location. These tasks are often more difficult for people with disabilities. For finding, navigation, and orientation, it is important that the user can find out what the current location is. For navigation, information about the possible destinations needs to be available. Screen readers convert content to synthetic speech which, because it is audio, must be presented in linear order. Some Success Criteria in this guideline explain what provisions need to be taken to ensure that screen reader users can successfully navigate the content. Others allow users to more easily recognize navigation bars and page headers and to bypass this repeated content. Unusual user interface features or behaviors may confuse people with cognitive disabilities.

Navigation has two main functions:

* to tell the user where they are
* to enable the user to go somewhere else

This guideline works closely with [Guideline 1.3](https://www.w3.org/WAI/WCAG22/Understanding/adaptable), which ensures that any structure in the content can be perceived, a key to navigation as well. Headings are particularly important mechanisms for helping users orient themselves within content and navigate through it. Many users of assistive technologies rely on appropriate headings to skim through information and easily locate the different sections of content. Satisfying [Success Criterion 1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships) for headings also addresses some aspects of Guideline 2.4.

**Comment by maryjom** [comment]
##Updated in WCAG 2.1 was the name of Guideline 4.1

###Guideline 4.1: Compatible
NOTE: There were no other changes to this guideline.

**Comment by pday1** [comment]
> ### Guideline 2.5: Input Modalities

I think that this needs a note or caveat - closed systems may not support 3rd party input devices due to architectural or security reasons, and so would be excluded.

**Comment by patrickhlauke** [comment]
> I think that this needs a note or caveat - closed systems may not support 3rd party input devices due to architectural or security reasons, and so would be excluded.

They may not allow 3rd party ones, but the actual system may provide different input modalities yet expect only a specific input to be used for a specific control/task, and it should instead allow for all its provided modalities to be used

**Comment by maryjom** [comment]
@pday and @patrickhlauke I think some confusion is being introduced because the WCAG2ICT Note includes the intent for the various guidelines and Success Criteria - completely unchanged. WCAG2ICT is simply analyzing the actual text of the Guideline to see if it is applicable as-is, or if there needs to be any clarification of the Guideline text due to the use of web-based language. The scope of the work is not to modify or clarify the intent - even if it is written in web-based language. Any specific SCs that have caveats for non-web software or non-web documentation will be handled with the specific SC (soon to come work).

**Comment by maryjom** [comment]
I added Guideline 2.5 per our meeting discussion on 1 December. The changes for guidelines 2.3 and 2.4 will automatically get incorporated once we get the includes from WCAG working in the markdown files.

**Comment by maryjom** [comment]
Oops, forgot this is going to AG WG for review - the next step. Reopening.

**Comment by maryjom** [comment]
AG WG has a [survey](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-First-AG-review-4-sc/) open until 9 March with AG WG survey results discussion at a TBD future date (after CSUN conference).

**Comment by maryjom** [comment]
Complete and merged. Approved in the AG WG meeting on 21 March 2023.
